README.md

```diff
@@ -337,6 +337,21 @@ INVALID_EMAIL,bob
 
 The error returned should be: `{ "E-Mail" => "is invalid" }`
 
+### Complex quoting (Illegal quoting problem)
+
+```ruby
+CSV.parse("1,Glk,Lev \"The Black Spider\" Yashin,USSR")
+  # => CSV::MalformedCSVError: Illegal quoting in line 1.
+```
+
+The problem is that the CSV comma-delimited text should entirely surround by `"`
+not in part or do not contain `"` at all. You can pass `quote_char` option.
+
+```ruby
+CSV.parse("1,Glk,Lev \"The Black Spider\" Yashin,USSR", quote_char: "\x00")
+  # => [ ["1", "Glk", "Lev \"The Black Spider\" Yashin", "USSR"] ]
```

Comment:
You set the quote char: `\x00` which is a NULL character. This character is not likely to be present in the CSV file that your client outputs so the CSV parser behaves as if the quote chars were disabled.

I would rather use an example where the quote char is a single quote. It would be easier to understand for the developers. Could you also give an example that uses `CSVImporter` instead of `CSV`?